global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.100",
+    "version": "6.0.403",
```

Comment:
I am currently working on a pull request to support .NET 7.
As .NET 7 has quite some additions to the file system, I think it is important to keep up-to-date.